sketcherMinimizer.cpp

```diff
@@ -3607,4 +3609,93 @@ int sketcherMinimizer::morganScores(const vector<sketcherMinimizerAtom*>& atoms,
     return n;
 }
 
+// interactions with m_minimizer and m_fragmentBuilder
+std::vector<sketcherMinimizerStretchInteraction*> sketcherMinimizer::getStretchInteractions()
+{
+    return m_minimizer.getStretchInteractions();
+}
+
+std::vector<sketcherMinimizerInteraction*> sketcherMinimizer::getInteractions()
+{
+    return m_minimizer.getInteractions();
+}
+
+std::set<sketcherMinimizerAtom*> sketcherMinimizer::getChetoCs(
+    const std::vector<sketcherMinimizerAtom*>& allAtoms)
+{
+    return m_minimizer.getChetoCs(allAtoms);
+}
+
+std::set<sketcherMinimizerAtom*> sketcherMinimizer::getAminoNs(
+    const std::vector<sketcherMinimizerAtom*>& allAtoms)
+{
+    return m_minimizer.getAminoNs(allAtoms);
+}
+
+std::set<sketcherMinimizerAtom*> sketcherMinimizer::getAlphaCs(
+    const std::vector<sketcherMinimizerAtom*>& allAtoms,
+    const std::set<sketcherMinimizerAtom*>& chetoCs,
+    const std::set<sketcherMinimizerAtom*>& aminoNs)
+{
+    return m_minimizer.getAlphaCs(allAtoms, chetoCs, aminoNs);
+}
+
+void sketcherMinimizer::clearInteractions()
+{
+    m_minimizer.clearInteractions();
+}
+
+void sketcherMinimizer::addInteractionsOfMolecule(
+    sketcherMinimizerMolecule* molecule, bool intrafragmentClashes)
+{
+    m_minimizer.addInteractionsOfMolecule(molecule, intrafragmentClashes);
+}
+
+void sketcherMinimizer::minimizeMolecule(sketcherMinimizerMolecule* molecule)
+{
+    m_minimizer.minimizeMolecule(molecule);
```

Comment:
@ZontaNicola should `sketcherMinimizer::minimizeMolecule()` call `CoordgenFragmenter::splitIntoFragments()`? When would someone not want the call to `splitIntoFragments`?

I'm asking based on my reading of https://github.com/rdkit/rdkit/blob/master/External/CoordGen/CoordGen.h#L220

Reply:
yes, you're right. Assuming that we want users to be able to call this function at all, it's gonna crash without splitIntoFragments first.